### DISCUSSION TITLE
Looking for new maintainers

### DISCUSSION THREAD

**Discussion by orhun** [ratatui/ratatui#151] [Discussion]
`ratatui` needs more love thus more people to take care of it. For now, me, @sayanarijit and @mindoodoo are responsible for reviewing the pull requests, organizing meetings and creating releases. However, as you might have guessed, we are not always enough for a project at this scale. So we need more developers to take care of the following tasks:

- Reviewing pull requests for quality code.
- Maintaining the codebase.
- Creating issues & release milestones.
- Providing comments & help.

If you are interested in maintaining `ratatui`, feel free to comment below and we're happy to consider things! 🐭 

P.S. For this role, it would be nice to be present on Discord, but it is not strictly needed.

**Comment by orhun** [comment]
I would like to propose @sophacles and @joshka for the maintainer role since they are actively contributing and providing code review for the pull requests.

Let us know if you would like to volunteer!

**Comment by lthoerner** [comment]
I'm not sure whether I have the skills to do it but I would be interested in getting involved if you guys need me.

**Comment by joshka** [comment]
I'm happy to help out with PRs etc., but probably not too much on discord.

**Reply by mindoodoo** [reply]
Yeah to be fair help with PRs is the main thing we need so... awesome ! Are you on the discord server ? (Just so we get in touch with you)

edit: Way too many typos lmao

**Reply by orhun** [reply]
Awesome, what is your preferred way of communication? We can work something out.

**Reply by joshka** [reply]
I have discord and joined the Ratatui server. Happy to use it sporadically, but I'm just well aware of what a time suck (for my brain) idling on slack/discord/etc. can be. If there was maybe 1 or two channels, perhaps it would be ok, but there's 20 or so, which I found a bit overwhelming.

I'd much prefer to see support type issues end up being public where possible (discussions / issues / PRs instead of discord messages) as it means that future users benefit from the answers etc whereas discord generally has a discoverability problem.

**Reply by Nickiel12** [reply]
It is certainly better for documentation/future bugs to have them on GitHub where anyone can see them, but the response time in conversations is a lot longer on GitHub. It might be worth looking into tools that will create issues and automatically log discord messages to GitHub issues? Discord thread=issue? Maybe?

But, right now discord seems to be the main hub of communication, but GitHub is the secondary. We are encouraging all users with actual crate issues to make GitHub issues though.

**Reply by orhun** [reply]
Just sent you a collaborator invite @joshka

**Comment by sophacles** [comment]
I'd be happy to help out!

(Sorry for the slow reply I was taking long weekend.)

**Reply by orhun** [reply]
Sent you a collaborator invite!

**Comment by kdheepak** [comment]
I would also like to help out in any way I can!

**Comment by joshka** [comment]
Hey @Valentin271 and @a-kenji would you be interested in being added to the maintainer's list?

**Reply by Valentin271** [reply]
Gladly yes! I'm already on the discord server.

**Reply by a-kenji** [reply]
I am also willing to contribute, tough my bandwidth varies. I have grown fond of the project and want to help out.

**Reply by joshka** [reply]
Sweet - we're mainly looking for a few extra eyes on reviews when you can spare some time. Many hands make light work.

**Reply by orhun** [reply]
Hey @a-kenji, are you on Discord? What is the best way to reach out if there is anything urgent?

**Comment by orhun** [comment]
I think it is time to bump this issue (again). We're looking for new rats! 🧀🐁 

With the growing popularity of Ratatui and recent [ecosystem projects](https://github.com/ratatui/ratatui/discussions/2367) addition to the organization, the maintenance responsibility and workload got higher, as expected. On top of that, our current maintainers seem to have limited availability these days which makes moving forward slower. 

So if you are looking for an opportunity to start contributing to your favorite TUI library in Rust, it's time! We are essentially looking for rats to triage issues/milestones, test/review PRs and overall help with the direction of the project. [Discord](https://discord.gg/pMCEU9hNEj) is what we use primarily for communication, but feel free to leave a comment here if you are interested.

**Reply by junkdog** [reply]
i'm already in the neighborhood; i can review stuff etc in ratatui too.

**Reply by gold-silver-copper** [reply]
+1 for junkdog

I would be happy to help maintain as well, especially in regards to bevy_ratatui

**Reply by joshka** [reply]
+1 to both these guys

**Comment by JayanAXHF** [comment]
I can help with reviews, docs and the website (and templates too)! Would love to be able to make Ratatui even better

**Reply by joshka** [reply]
+1 I think

**Comment by Vaishnav-Sabari-Girish** [comment]
I can help with reviews and Embedded Rust work. I’d be happy to contribute to `ratatui`
